### ISSUE TITLE
Update the screenshots in the README

### ISSUE THREAD

**Issue by pschmitt** [zdharma-continuum/zinit#84]
Right now, we feature a bunch of outdated screenshot that were taken before the rebranding to zinit and the "deletening".
The zplugin days are long over!

Can someone please take some glamour shots? Thanks!

**Comment by alichtman** [comment]
![image](https://user-images.githubusercontent.com/20600565/142568554-6dde6685-3133-4de2-95be-b7f4b2047b5e.png)
![image](https://user-images.githubusercontent.com/20600565/142568682-ca2e34fb-08d7-4c0d-8f68-4b99ef1f4f85.png)
![image](https://user-images.githubusercontent.com/20600565/142568703-7798889f-3406-4b33-8ea1-b94d34374f89.png)

**Comment by alichtman** [comment]
Here's some screenshots. Not a HUGE fan of any in particular.

**Comment by pschmitt** [comment]
Thanks for those. I don't think this will work due to the resolution.

Since we live in a smartphone works we need bigger pictures ie fonts 😉

I'll try to fetch some screens off of Termux if I find the time.

**Comment by alichtman** [comment]
> Since we live in a smartphone works 

yeah but I don't think people are seriously working in terminals on their phones. It's ok to make the pics mobile-unfriendly

**Comment by pschmitt** [comment]
> yeah but I don't think people are seriously working in terminals on their phones

I am :kissing_heart: (yes, really.) Most of my contributions to zinit were made from within Termux :man_shrugging: 

>  It's ok to make the pics mobile-unfriendly

The web is first and foremost for smartphones now. It's a long-lost battle. 
Having a readable screenshot benefits both regular computer and mobile users.

**Comment by vladdoster** [comment]
@pschmitt Closing due to lack of activity.
